src/.prettierrc.toml

```diff
@@ -0,0 +1,4 @@
+trailingComma = "es5"
```

Comment:
I see multiple configurations for `prettier`..have one file.

Reply:
removed